### PR TITLE
/getSpaceUsage, /login and /createPassword apis

### PR DESCRIPTION
**Pull request by mawnika**

### CODE REVIEW

File: src/main/java/com/nasrpi/auth/AuthController.java [hunk 1]
```diff
@@ -29,5 +30,15 @@
 	public boolean authenticate() {
```
Comment (zuilee) [inline src/main/java/com/nasrpi/auth/AuthController.java:30]: what is this method?

Comment (mawnika) [reply src/main/java/com/nasrpi/auth/AuthController.java:30]: This you had written to check if the config file exists or not and to show the appropriate button (login or create password)

Comment (zuilee) [reply src/main/java/com/nasrpi/auth/AuthController.java:30]: alright

File: src/main/java/com/nasrpi/home/GetSpaceModel.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+package com.nasrpi.home;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
```
Comment (zuilee) [inline src/main/java/com/nasrpi/home/GetSpaceModel.java:12]: please make necessary changes

File: src/main/java/com/nasrpi/home/HomeRepository.java [hunk 3]
```diff
@@ -160,4 +160,24 @@ public boolean moveFile(String source, String destination) {
 
 	}
 
-}
+	public List<GetSpaceModel> getSpaceUsage(String rootPath) {
```
Comment (zuilee) [inline src/main/java/com/nasrpi/home/HomeRepository.java:163]: what does this method return exactly?

File: src/main/java/com/nasrpi/home/GetSpaceModel.java (hunk 2 above)
Comment (mawnika) [reply src/main/java/com/nasrpi/home/GetSpaceModel.java:12]: what changes?

**Review by zuilee — APPROVED** [review]
Please update the GetSpaceModel class author in your nest pr. I am merging this one for now.

**Pull request merged**